inbm/dispatcher-agent/dispatcher/sota/snapshot.py

```diff
@@ -41,11 +41,12 @@ class Snapshot(metaclass=ABCMeta):  # pragma: no cover
     """
 
     def __init__(self, trtl: Trtl, sota_cmd: str,  snap_num: Optional[str],
```

Comment:
Ensure that the `device_reboot` parameter is documented in the docstring of the `Snapshot` class constructor to maintain consistency and clarity. [important]